classroom-dashboard/app/assets/stylesheets/application.css

```diff
@@ -25,6 +25,8 @@
 @import "bootstrap";
 @import url('https://fonts.googleapis.com/css?family=Montserrat|Work+Sans:300');
 
+// should this be restructured a certain way so its more intuitive?
+
 
```

Comment:
I think our selectors/class names can be restructured in a way that makes it easier to locate a specific one if necessary. I think we can some of them so that we are able to tell which file/webpage the selector refers to.

Right now, for example, we have an h1 that spans about two or three files - however, if we ever want to change the declarations for a single one on a specific webpage, it would change them all, which might not be necessarily what we want.

We could go through all of our files and add individual names/classes to our selectors, as well as leave comments above each section, to make this process a little smoother!